setup.cfg

```diff
@@ -39,30 +39,28 @@ setup_requires =
   setuptools-declarative-requirements
 install_requires =
   salt>=3002
-  jinja2<=3.0.1
-  azure-batch==4.1.3
-  azure-common==1.1.10
-  azure-graphrbac==0.40.0
-  azure-keyvault==0.3.7
-  azure-mgmt-authorization==0.30.0
-  azure-mgmt-batch==5.0.0
-  azure-mgmt-compute==3.1.0rc3
-  azure-mgmt-dns==1.2.0
-  azure-mgmt-keyvault==0.40.0
-  azure-mgmt-msi==0.1.0
-  azure-mgmt-network==2.0.0rc1
-  azure-mgmt-resource==1.2.2
-  azure-mgmt-storage==1.5.0
-  azure-mgmt-subscription==0.1.0
-  azure-mgmt-web==0.34.1
-  azure-storage==0.36.0
-  azure-storage-blob==1.1.0
-  azure-storage-common==1.1.0
-  azure-storage-file==1.1.0
-  azure-storage-nspkg==3.0.0
-  azure-storage-queue==1.1.0
-  msrest==0.4.25
-  msrestazure==0.4.25
+  jinja2<=3.1.2
+  azure-batch==12.0.0
+  azure-common==1.1.28
+  azure-graphrbac==0.61.1
+  azure-keyvault==4.2.0
+  azure-mgmt-authorization==2.0.0
```

Comment:
**todo (blocking):** I still don't see the new `azure-identity` and `azure-mgmt-privatedns` dependencies. Check to ensure any other new imports are covered as well.